### PR TITLE
Gh1262 exists

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] Rollback provided (if necessary)?
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/doc/ltnews39.tex [hunk 1]
```diff
@@ -373,6 +374,13 @@ \subsection{New conditionals: \cs{IfClassAtleastT} and friends}
 that one can only test for files that contain a \cs{ProvidesFile}
 line.
 %
+We did the same for the conditionals \cs{IfLabelExistsTF} and
+\cs{IfPropertyExistsTF} also introduced in 2023.\footnote{By mistake they
+were initially introduced under the names \cs{IfLabelExistTF} and
+\cs{IfPropertyExistTF}; we corrected that at the same time. This is a
+breaking change, but the commands haven't yet been used so far only in
+kernel code.}
```
Comment (josephwright) [inline base/doc/ltnews39.tex:382]: Did you mean 'the commands haven't been used so far outside of kernel code'?

File: base/ltproperties.dtx [hunk 2]
```diff
@@ -746,9 +746,13 @@
 %    \end{macrocode}
 % \end{macro}
 %
-% \begin{macro}{\IfPropertyExistTF}
+% \begin{macro}{\IfPropertyExistsTF,\IfPropertyExistsT,\IfPropertyExistsF}
+% \changes{v1.0e}{2024-04-17}{Reanmed \cs{IfPropertyExistTF} to
```
Comment (josephwright) [inline base/ltproperties.dtx:750]: 'Renamed'

File: base/ltproperties.dtx [hunk 3]
```diff
@@ -746,9 +746,13 @@
...
+%                             \cs{IfPropertyExistsTF} (gh/1262)} 
 %    \begin{macrocode}
-\cs_new_eq:NN \IfPropertyExistTF \property_if_exist:eTF  
+\cs_new_eq:NN \IfPropertyExistsTF \property_if_exist:eTF  
+\cs_new:Npn   \IfPropertyExistsT #1#2 {\property_if_exist:eTF {#1}{#2}{} }  
+\cs_new:Npn   \IfPropertyExistsF #1   {\property_if_exist:eTF {#1}{} }
```
Comment (josephwright) [inline base/ltproperties.dtx:755]: Do we not have `expl3` level `\property_if_exist:eT`, etc?

**Review by josephwright — CHANGES_REQUESTED** [review]

File: base/ltproperties.dtx (hunk 3 above)
Comment (muzimuzhi) [reply base/ltproperties.dtx:755]: Ah yes, only `TF` is generated.
https://github.com/latex3/latex2e/blob/a46f5273447b9a26b518272ec5a510c2c6136ed5/base/ltproperties.dtx#L745

Comment (josephwright) [reply base/ltproperties.dtx:755]: Probably should change that then - I think it's odd to have the CamelCase ones but not the `expl3` ones

Comment (josephwright) [reply base/ltproperties.dtx:755]: OK, I can live with that :)

**Review by josephwright — APPROVED** [review]

**Review by davidcarlisle — APPROVED** [review]

**Pull request merged**